ethcore/src/block.rs

```diff
@@ -405,11 +405,13 @@ impl Drain for SealedBlock {
 	}
 }
 
-/// Enact the block given by block header, transactions and uncles
+/// Enact the block. Takes the block header, transactions and uncles from a
+/// `PreVerified` block and Produces a new `LockedBlock` after applying all
+/// transactions and committing the state to disk.
 pub(crate) fn enact(
-	header: Header,
-	transactions: Vec<SignedTransaction>,
-	uncles: Vec<Header>,
+	header: &Header,
+	transactions: &Vec<SignedTransaction>,
```

Comment:
when this PR is ready, don't forget to change `&Vec<_>` params to `&[_]`
https://rust-lang.github.io/rust-clippy/master/index.html#ptr_arg

Reply:
Cool, TIL! :)